This backports 0ccc09a4fca700a33055fc93e762ed0ca134a7aa which notes that the
linux-loader and vm-memory crates must be kept in sync.

--- a/src/utils/Cargo.toml
+++ b/src/utils/Cargo.toml
@@ -13,7 +13,7 @@
 versionize = "0.1.10"
 versionize_derive = "0.1.5"
 vmm-sys-util = "0.11.0"
-vm-memory = { version = "0.10.0", features = ["backend-mmap", "backend-bitmap"] }
+vm-memory = { version = "0.12.0", features = ["backend-mmap", "backend-bitmap"] }
 
 net_gen = { path = "../net_gen" }
 
--- a/src/vmm/Cargo.toml
+++ b/src/vmm/Cargo.toml
@@ -13,7 +13,7 @@
 kvm-ioctls = "0.14.0"
 lazy_static = "1.4.0"
 libc = "0.2.117"
-linux-loader = "0.8.1"
+linux-loader = "0.9.0"
 log = "0.4.17"
 rand = "0.8.5"
 serde = { version = "1.0.136", features = ["derive"] }
